backend/corpora/lambdas/api/v1/curation/collections/actions.py

```diff
@@ -40,3 +41,12 @@ def get(visibility: str, token_info: dict, curator: str = None):
         resp_collections.append(resp_collection)
 
     return jsonify(resp_collections)
+
+
+def post(body: dict, user: str):
+    try:
+        return create_collection(body, user)
+    except InvalidParametersHTTPException as ex:
+        ex.ext = dict(invalid_parameters=ex.detail)
```

Comment:
I'm a little confused. The `ex` instance of an `InvalidParametersHTTPException` is already going to have a `.detail` attribute on it, no? So we're putting the `errors` array on both `ex.detail` and `ex.ext`??

Reply:
We are putting  `InvalidParametersHTTPException._detail` in `ex.detail`. See the next line down.

Reply:
Shouldn't the next line down be `ex._detail`?

Or is this `_detail` attribute a _class_ attribute/detail?

Reply:
it's a class attribute, which is why is why put the `_` prefix. I'll make it`__` since it's a private class attribute.

Reply:
actually, the convention is still to a single under-score prefix for private class variables. but I updated the variable name to make it more obvious what its purpose is./